### PR TITLE
[Storage] add CID to bucket item row

### LINKED ISSUE #1420: Show file item CID's for bucket contents
**Is your feature request related to a problem? Please describe.**
The CID of each file uploaded bucket should be made visible to the user

**Describe the solution you'd like**
Either display this value directly in the table, or alternatively, add the File Info Modal that can be brought up via context menu on each line item.

image
 or 
 
image

### PR DESCRIPTION
**Pull request by FSM1**
Closes #1420

### CODE REVIEW

File: packages/storage-ui/src/Components/Modules/FilesList/FilesList.tsx [hunk 1]
```diff
@@ -779,15 +781,14 @@ const FilesList = () => {
...
-                  align="left"
                   onSortChange={() => handleSortToggle("name")}
                   sortDirection={column === "name" ? direction : undefined}
                   sortActive={column === "name"}
                 >
                   <Trans>Name</Trans>
                 </TableHeadCell>
                 {
-                  fileSystemType && fileSystemType !== "ipfs" && <TableHeadCell
+                  desktop && fileSystemType && fileSystemType !== "ipfs" && <TableHeadCell
```
Comment (Tbaut) [inline packages/storage-ui/src/Components/Modules/FilesList/FilesList.tsx]: No need for desktop check here (thanks lgtm-bot)
```suggestion
                  fileSystemType && fileSystemType !== "ipfs" && <TableHeadCell
```

File: packages/storage-ui/src/Components/Modules/FilesList/FilesList.tsx [hunk 2]
```diff
@@ -799,6 +800,11 @@ const FilesList = () => {
                     <Trans>Date uploaded</Trans>
                   </TableHeadCell>
                 }
+                {
+                  desktop && <TableHeadCell>
+                    CID
+                  </TableHeadCell>
+                }
```
Comment (Tbaut) [inline packages/storage-ui/src/Components/Modules/FilesList/FilesList.tsx]: Same here (thanks lgtm-bot bis)
```suggestion
                <TableHeadCell>
                    CID
                </TableHeadCell>
```

**Review by Tbaut — APPROVED** [review]
Looks good, just the 2 warning from lgtm-bot. Those conditions are already in a `{desktop && ...}` placed above.

**Review by tanmoyAtb — APPROVED** [review]
looks good!

**Pull request merged**